optin/puller/tests/test/OptinRequestTestUtils.py

```diff
@@ -0,0 +1,48 @@
+from symbolchain.CryptoTypes import Hash256
+from symbolchain.nem.Network import Address
+
+from puller.models.OptinRequest import OptinRequest, OptinRequestError
+
+HEIGHTS = [
+	12345678905, 12345678901, 12345678903
+]
+
+
+HASHES = [  # sorted
+	'FA650B75CC01187E004FCF547796930CC95D9CF55E6E6188FC7D413526A840FA', 'C16CF93C5A1B6620F3D4C7A7EAAE1990D9C2678775FBC18EAE577A78C8D52B25',
+	'7D7EB08675D6F78FAB8E7D703994390DD95C6A9E8ADD18A9CF13CE4C632F8F01'
+]
+
+
+PUBLIC_KEYS = [
+	'5C80309DD937ACB0869F1BEEDE14B552E02AEC4B3569E2E2E76477F84B7E3121', '5AA84DF43CD5173A79D91310CE2042434F8B4A7DA157CE3C9DE5C993AEC18293',
+	'E3CCB9628EA0CE412BAF7E91B215E0645DC6B83D8CCEB3E849E51A3CF15B0A2F', '02028714E352FE1348AC40977F23B1954EAC5836B0BEDD2A5F26E34A83DB773D',
+	'AF39DF4A01DA4DDB3545CB1694F71C5D0EB062048DA13ABA871D34BDB04BA715'
+]
+
+
+NEM_ADDRESSES = [
+	'TAHTNAEQNDJOBDHHRON7SKU7PO6GAWXAJZ4CB2QG', 'TA4FO6TJBIG3VODLUJTPYTJPXDOSTOFZKDGMXPZF', 'TA7TKNUYFE5BVW6HELM7GAUECYPQ6ATK5MR4DNQ5',
+	'TB7GF436SYPM4UQF2YYI563QIETUO5NZR6EREKPI', 'TCYIHED7HZQ3IPBY5WRDPDLV5CCMMOOVSOMSPD6B'
+]
```

Comment:
should public keys and addresses match?

addresses were generated from these public keys:

	'C0EB403A0EF1949D656EA775FCC7BFA6D8288E70AC831847686D56E6BB3D445C', 'A52DF1C5AD709689EDF75650CD292B55678ACBCE3F17D300D16B680CEB648D41',
	'55C3979A27EBF358496E673BE7C39257894B754BE92ED6456314DE9FAA2D447D','138F8ECE0F01DC7CCD196F2C6249CBB78CF2822D23376C96C949DF859D5A0FC5',
	'1BB7ACCACE4C2527F425B2156C676105DF013404F7FA1F169377CA05393AECBE',